pkg/restclient/volume.go

```diff
@@ -29,9 +29,9 @@ func (c *Client) Volumes(volname string) (api.VolumeListResp, error) {
 }
 
 // VolumeStatus returns the status of a Gluster volume
-func (c *Client) VolumeStatus(volname string) (api.VolumeStatusResp, error) {
-	url := fmt.Sprintf("/v1/volumes/%s/status", volname)
-	var volStatus api.VolumeStatusResp
+func (c *Client) VolumeStatus(volname string) (api.BricksStatusResp, error) {
```

Comment:
Volume status endpoint still exists, and this needs to still match it. Add a new function for brick status.